openreview/tools.py

```diff
@@ -1969,11 +1970,7 @@ def get_neurips_profile_info(profile, n_years=3):
         if email.startswith("****@"):
             raise openreview.OpenReviewException("You do not have the required permissions as some emails are obfuscated. Please login with the correct account or contact support.")
         emails.add(email)
-
-    ## if institution section is empty, add email domains
-    if not domains:
-        for email in profile.content['emails']:
-            domains.update(openreview.tools.subdomains(email))
+        domains.update(openreview.tools.subdomains(email))
```

Comment:
why did you make this change?

I think this is part of the NeurIPS policy, only take into account domain emails when the history doesn't have any domains. @sherry-zyx please confirm.

Reply:
I think it misses conflicts otherwise. For example, given `profile1` and `profile2`:
```
        profile1 = openreview.Profile(
            id = 'Test_Conflict1',
            content = {
                'emails': ['user@cmu.edu'],
                'history': [{
                    'institution': {
                        'domain': 'user@126.com'
                    }
                }]
            }
        )

        profile2 = openreview.Profile(
            id = 'Test_Conflict2',
            content = {
                'emails': ['user2@126.com'],
                'history': [
                    {
                        'institution': {
                            'domain': 'user2@cmu.edu'
                        }
                    }]
            }
```
they won't have conflicts with the original code

Reply:
because they don't share any domain. The domains must be only urls not email addresses. `126.com` is skipped I believe.